.buildkite/scripts/test_e2e.sh

```diff
@@ -28,11 +28,18 @@ if [[ "${BUILDKITE:-""}" != "" ]]; then
     mkdir -p ${WORKDIR}/e2e
 fi
 
+# Use integrationrunner-wrapper.sh as node binary if we need to compute E2E
+# tests' coverage.
+node_binary=""
```

Comment:
Hm, default is no binary? That won't work?

Reply:
Yea, submitted the PR too fast, fixed it :slightly_smiling_face: